CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
 ### Added
 - Added signatures for `cds.outboxed` and `cds.unboxed`
+- Added signatures for `cds.middlewares`
```

Comment:
Actually these are signatures for `cds.middlewares.add`.
There are other methods for [cds.middlewares](https://cap.cloud.sap/docs/node.js/cds-serve#cds-middlewares) . 
Are they out of scope of this PR ?

Reply:
We have only identified `.add` as a relevant method in our preliminary sync. If you feel like there are others that should be typed, then they can surely be added to this PR!

Reply:
Still, you are right to object to the changelog message. I have updated it accordingly.